iib/workers/tasks/build.py

```diff
@@ -703,3 +707,68 @@ def handle_regenerate_bundle_request(from_bundle_image, organization, request_id
         'state_reason': 'The request completed successfully',
     }
     update_request(request_id, payload, exc_msg='Failed setting the bundle image on the request')
+
+
+def _copy_files_from_image(image, src_path, dest_path):
```

Comment:
Do we want to add the `@retry` decorator here?

Reply:
That might mean splitting the method though

Reply:
It might be tricky to distinguish whether or not retry should be performed. For instance, if the image provided doesn't have the `/manifests` directory, it would fail but we shouldn't retry.

Can we leave it as is and address this in the future if it does become a problem?

Reply:
Since the image is not pulled down at this point, it's possible that this would fail, especially if the image is pulled through the registry proxy. Perhaps you could just pull down the image beforehand using retries and then keep this function as is?

Reply:
Cool, I can live with that. :)